src/components/Icon.tsx

```diff
@@ -1,5 +1,13 @@
 import * as React from "react";
-import { Image, Text, View, StyleSheet } from "react-native";
+import {
+  Image,
+  Text,
+  View,
+  StyleSheet,
+  ViewProps,
+  StyleProp,
+  ImageStyle,
+} from "react-native";
 // import { Ionicons } from "@expo/vector-icons";
 // import * as VectorIcons from "@expo/vector-icons";
 // console.log("VECTORICONS", VectorIcons);
```

Comment:
@peterpme I can't remember why we kept this, if you can't either we should probably just remove it at this point.